internal/pkg/service/stream/mapping/recordctx/http.go

```diff
@@ -84,6 +84,7 @@ func (c *httpContext) HeadersMap() *orderedmap.OrderedMap {
 }
 
 func (c *httpContext) ReleaseBuffers() {
+	_ = c.req.Body.Close()
```

Comment:
This is more of a guess but shouldn't we close the request body `io.ReadCloser` here? We're not planning to read from it again. Fast http internally does this (and ignores the err).

Reply:
I don't think so if you want to reuse the buffers. It puts the body into the pool. If you do this you will reuse closed body and need to recreate the req.Body.
We can try it but I think this is not good approach. Could you test it locally with k6? You should have memory leak in the stream with previous version and some better performance .
https://github.com/keboola/keboola-as-code/blob/main/docs/stream/benchmarks.md

Reply:
I think the comment is unrelated to this line? `net/http` doesn't seem to be using any pool in the first place, only fasthttp does.